pnpm-lock.yaml

```diff
@@ -31595,6 +31595,7 @@ packages:
   /clone@1.0.4:
     resolution: {integrity: sha512-JQHZ2QMW6l3aH/j6xCqQThY/9OH4D/9ls34cgkUBiEeocRTU04tHfKPBsUK1PqZCUQM7GiA0IIXJSuXHI64Kbg==}
     engines: {node: '>=0.8'}
+    requiresBuild: true
```

Comment:
the changes to the pnpm-lock file seem unrelated to this PR, and should probably be removed from it.